### ISSUE TITLE
IE11 Support Remains in v2

### ISSUE THREAD

**Issue by Malix-off** [bigskysoftware/htmx#2637]
## Context

- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/www/content/posts/2024-06-17-htmx-2.0.0-is-released.md?plain=1#L8-L11

- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/www/content/migration-guide-htmx-1.md#L55-L56

## Remains

_Permalink URLs to text, comments, and entities containing "IE"_
_I did not included all the mentions of "IE11" yet_

### Documentation

- https://github.com/bigskysoftware/htmx/blob/c6a89b315b6f964f6d69fa53252c2eb3e63aa8c8/README.md?plain=1#L21
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/CONTRIBUTING.md#L15

### `src/` and `test/`

- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/src/htmx.js#L773
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/src/htmx.js#L853
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/src/htmx.js#L1094
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/src/htmx.js#L1563
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/src/htmx.js#L1615
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/src/htmx.js#L1617
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/src/htmx.js#L3075
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/src/htmx.js#L3807
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/src/htmx.js#L3891
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/test/core/ajax.js#L779
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/test/core/api.js#L241
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/test/core/shadowdom.js#L36
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/test/core/shadowdom.js#L36
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/test/core/shadowdom.js#L841
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/test/util/util.js#L14

### `www/content/`

- https://github.com/bigskysoftware/htmx/blob/c6a89b315b6f964f6d69fa53252c2eb3e63aa8c8/www/content/_index.md?plain=1#L109
- https://github.com/bigskysoftware/htmx/blob/685bbdf58bcdf563d60352129662deeae3d1fa4a/www/content/essays/no-build-step.md#L48-L54

**Comment by Malix-off** [comment]
@Telroshan it's not just about documentation, see [`src/` and `test/`](https://github.com/bigskysoftware/htmx/issues/2637#:~:src/%20and%20test/,-htmx/src/htmx)

**Comment by Telroshan** [comment]
I get that, though the IE11 fallbacks are minor at this point ; dropping IE11 support just means we won't actively maintain IE11 compatibility anymore, and if something breaks on IE11 only, it's considered "normal". Having IE11 fallback still in isn't an issue in itself, and the only advantage I see here by removing them altogether would be to reduce the lib size and spare some additional bytes (which, don't get me wrong, is a valid argument, but pretty minor considering the amount of code we're talking about)

**Comment by 1cg** [comment]
Fixed in docs, closing this issue.  IE references will be slowly removed over time in src and test, no need for an issue around it.
